### PR TITLE
Index settings tutorials

### LINKED ISSUE #2634: Index settings in Meilisearch Cloud
While conducting interviews for other features, some Meilisearch employees mentioned it could be good for the documentation to include instructions on how to manage settings in Meilisearch Cloud.

Further research is necessary to determine how much user demand there is for this, and what content we need to include in a guide.

## Data collection
- [ ] User feedback (complaints, support tickets, social media/discord discussions)
- [ ] Interview product designer about intended workflow and potential issues

## Questions
- There are plenty of index settings. Is it worth to walk through each one in the interface?
- Instead of a new guide, would it make more sense to update the current API reference with a cloud-specific section?
  - This might mean breaking up the index settings reference, as the current page is already very long and hard to parse

### PR DESCRIPTION
**Pull request by guimachiavelli**
Closes #2634.

Instead of the two options listed in the issue, I have opted for creating a short tutorial using the Cloud interface. This solves the issue of documenting an increasing number of interface elements with almost identical usage.

Since many users will prefer to use the API instead of the interface, this guide also adds a second tutorial teaching users to do the exact same thing with the API.

## Notes for reviewers

- @Strift: as usual, all feedback is welcome, but if you're short on time my main request is that you take a look at the fathom URL params and see if they align with how dev rel is handling them.

### CODE REVIEW

**Review by Strift — COMMENTED** [review]
The UTMs parameters are correct. 👌 

You can add in the [UTM tracking sheet](https://www.notion.so/64b3356d2ce84d87b4fe22feb91f4d82?v=1612b9eee2e34352ab55ea64f45074ba) to document it.

Regarding the content, I think we should not encourage API usage. The API should be seen as a fallback—a way to convey that Meili works with any stack.

Instead, I think we should demonstrate that Meilisearch integrates easily with our ICP stack via the following options:
1. Via the UI (less tech work to do)
2. Via the JavaScript SDK (target ecosystem n°1)
3. Via the PHP SDK (target ecosystem n°2)
4. Other options

I hope this helps

File: learn/configuration/configuring_index_settings_api.mdx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+## Conclusion
+
+You have used the Meilisearch API to check the value of an index setting. This revealed an opportunity to improve your project's performance, so you updated this index setting to make your application better and more responsive.
+
+This tutorial used the searchable attributes setting, but the procedure is the same no matter which index setting you are editing.
+
+## What's next
+
+If you are Meilisearch Cloud user, you may also [configure index settings using the Meilisearch Cloud UI](/learn/configuration/configuring_index_settings).
```
Comment (Strift) [inline learn/configuration/configuring_index_settings_api.mdx:111]: Since "using the cloud" is the journey we're trying to promote, I would move this to the introduction of the article

**Comment by guimachiavelli** [comment]
Thanks for the feedback, @Strift!

I'm interested on the product team feedback on this too, but for the moment I want to clarify a few things.

If I understand correctly, by API you mean `curl`, right? If that's the case, no need to worry. I left the code samples in code blocks so it's easier to see them in context, but once I get the ok from the technical reviewers I will move them to the code samples component and notify integrations we need new samples for JS and PHP (and potentially other SDKs, eventually). `curl` will remain available as an option because we still want users to follow along without having to set up a whole environment.

**Comment by Strift** [comment]
Great. I think we'll have to handle the code samples ourselves since there isn't an integration team anymore. :p

**Review by macraig — DISMISSED** [review]
LGTM!

**Review by macraig — APPROVED** [review]

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -1158,3 +1158,17 @@ update_proximity_precision_settings_1: |-
...
+  curl \
+    -X PUT 'http://localhost:7700/indexes/INDEX_NAME/settings/searchable-attributes' \
+    -H 'Content-Type: application/json' \
+    --data-binary '[
+      "title",
+      "overview"
+    ]'
+index_settings_tutorial_api_task_1: |-
+curl \
+  -X GET 'http://localhost:7700/tasks/TASK_UID'
```
[the remark this answers is not in the document]
Comment (curquiza) [reply .code-samples.meilisearch.yaml:1174]: `TASK_UID` will not be a valid task uid (contrary to `INDEX_NAME` for the index uid), should we keep it as it is?

[the remark this answers is not in the document]
Comment (guimachiavelli) [reply .code-samples.meilisearch.yaml:1174]: I would prefer to keep it as is. I prefer a user sees an explicit error because they copypasted without replacing the placeholder, than the example including an arbitrary uid that might correspond to an unrelated task (e.g. the code sample uses uid `1` and `1` corresponds to some settings they updated before starting the tutorial).

**Review by macraig — APPROVED** [review]
